File: MangoAPI.DTO/ApiCommands/Messages/EditMessageCommand.cs
using MangoAPI.DTO.Responses.Messages;
using MediatR;

namespace MangoAPI.DTO.ApiCommands.Messages
{
    public record EditMessageCommand : IRequest<EditMessageResponse>
    {
        public string MessageId { get; set; }
        public string UserId { get; set; }
        public string ModifiedText { get; set; }
    }
}

File: MangoAPI.Client/src/app/types/requests/EditMessageCommand.ts
export class EditMessageCommand {
  messageId: string;
  chatId: string;
  modifiedText: string;

  constructor(messageId: string, chatId: string, modifiedText: string) {
    this.messageId = messageId;
    this.chatId = chatId;
    this.modifiedText = modifiedText;
  }
}
